it-and-security/lib/macos-device-health.policies.yml

```diff
@@ -17,18 +17,18 @@
   resolution: An an IT admin, deploy a macOS, login window profile with the DisableGuestAccount option set to true.
   platform: darwin
 - name: macOS - Require 10 character password
-  query: SELECT 1 WHERE 
```

Comment:
Why is it under `macOS - Require 10 character password` ?
Mistake ? 🙂

Reply:
@sharon-fdm whoops! Thanks for catching that.

Fixed this in my latest commit.